### DISCUSSION TITLE
Is the demo site down?

### DISCUSSION THREAD

**Discussion by CupcakeMunster** [netbox-community/netbox#10862] [Q&A]
Just tried logging in and nothing is populating. Tried from work and from home.

**Comment by jeremystretch** [comment]
Looks like someone was messing with the admin user. :roll_eyes: Fixed now. Thanks for the heads up.

**Reply by CupcakeMunster** [reply]
thank you. I was wondering if I could use the demo site to try and get my workplace on board with a presentation. I think I'll stand up my own demo server just in case.

**Reply by jeremystretch** [reply]
That's definitely a good idea.
